modules/sr/robot/motor_devices.py

```diff
@@ -23,10 +29,18 @@ def __init__(self, webot: Robot, motor_name: str) -> None:
         super().__init__(webot, motor_name)
         self.webot_motor.setPosition(float('inf'))
         self.webot_motor.setVelocity(0)
+        self.max_torque = self.webot_motor.getMaxTorque()
 
     def set_speed(self, speed: float) -> None:
         self.webot_motor.setVelocity(speed)
 
+        # If not using braking, set torque to 0.1 when motor speed is 0
+        # Otherwise set max torque
+        if(speed == 0 and not self._use_brake):
+            self.webot_motor.setAvailableTorque(0.1)
+        else:
+            self.webot_motor.setAvailableTorque(self.max_torque)
```

Comment:
As this is a method that we've not used before (also `setAvailableForce`), you may need to add it to the stubs in `stubs/controller.py`.

Reply:
I think they are already in there - do I need to change them at all?

Reply:
Oh, if they're already there then likely they don't need changing. (Feel free to double check them though)
I was assuming that this might be the cause of the type checking errors you're seeing on CI, but I hadn't clicked through to actually look at the errors.

Reply:
Looks like I was wrong. I should have checked the errors first, sorry about that!